### ISSUE TITLE
setting date_field does not trigger change event

### ISSUE THREAD

**Issue by ccdredzik** [watir/watir#938]
## Meta -
Watir Version: 6.19.1
Selenium Version: 3.142.7
Browser Version: 
Browser Driver Version: newest
OS Version: newest macOS

## Expected Behavior -

Changing the value of a date_field element will trigger the javascript change event.

## Actual Behavior -

Changing the value of a date_field element does not trigger the javascript change event.

## Steps to reproduce -

```
date_field :date_of_birth, { css: 'input[type=date]' }
...
self.date_of_birth = '1986-01-01' # this will not trigger the changed event
date_of_birth_element.fire_event('change') # workaround
```

**Comment by ccdredzik** [comment]
Rationale: setting a text_field in the same way does fire the change event. I would like for this behaviour to be consistent.

**Comment by titusfortner** [comment]
We switched to filling date fields by JavaScript instead of send_keys, since send_keys no longer worked. So, yeah, the change event wouldn't be fired... We can add that.

**Comment by ccdredzik** [comment]
@titusfortner thank you! Can I help in any way?

**Comment by titusfortner** [comment]
Feel free to make a PR with this code added. I'll probably not get to it until next week.

**Comment by titusfortner** [comment]
merged: 003b556ef8d5f5ce52908f661b04001399fe4481

Thanks for the suggestion.
